migrations/dimensions-config/bf6308af80aa.py

```diff
@@ -0,0 +1,402 @@
+            {
+                "name": "seq_end",
+                "type": "int",
+                "doc": "The sequence number of the final exposure of the visit that contains this exposure.",
+            },
+        )
+        exposure["metadata"].insert(
+            -1,
+            {
+                "name": "azimuth",
+                "type": "float",
+                "doc": (
+                    "Azimuth of the telescope at the start of the exposure in degrees. "
+                    "Can be NULL for observations that are not on sky, or for observations "
+                    "where the azimuth is not relevant."
+                ),
+            },
+        )
+        exposure["metadata"].append(
+            {
+                "name": "has_simulated",
+                "type": "bool",
+                "doc": (
+                    "True if this exposure has some content that was simulated. "
+                    "This can be if the data itself were simulated or if "
+                    "parts of the header came from simulated systems, such as observations "
+                    "in the lab that are recorded as on-sky."
+                ),
+            },
+        )
+
+        visit_system = config["elements"]["visit_system"]
+        visit_system["doc"] = (
+            "A system of self-consistent visit definitions, within which each "
+            "exposure should appear at most once.\n"
+            "A visit may belong to multiple visit systems, if the logical definitions "
+            "for those systems happen to result in the same set of exposures - the "
+            "main (and probably only) example is when a single-snap visit is observed, "
+            'for which both the "one-to-one" visit system and a "group by header metadata" visit '
+            "system will define the same single-exposure visit. "
+        )
+
+        visit_definition = config["elements"]["visit_definition"]
+        visit_definition["requires"].remove("visit_system")
+        visit_definition["requires"].append("visit")
+        del visit_definition["implies"]
+
+        config["elements"]["visit_system_membership"] = {
+            "doc": "A many-to-many join table that relates visits to the visit_systems they belong to.",
+            "requires": ["visit", "visit_system"],
+            "always_join": True,
+            "storage": {"cls": "lsst.daf.butler.registry.dimensions.table.TableDimensionRecordStorage"},
+        }
+
+        return config
+
+    _LOG.info("migrating stored dimesions.json")
+    attributes.update_dimensions_json(update_config)
+
+
+def _migrate_schema_digest():
+    """Update schema digest for dimension record tables.
+
+    Note that that new releases, right before dimensions manager version
+    6.0.2, stopped validating digests. We want to keep older releases happy
+    though, so this digest actually corresponds to dimensions manager 6.0.1
+    with updated dimensions configuration. And we only care about PostgreSQL.
```

Comment:
We will want to run migration on sqlite -- the digests were never used on sqlite then?

Reply:
Digests are used in sqlite too, I just though that we were always re-creating sqlite repos instead of migration them. I'll add checksum for sqlite too.

Reply:
Ah. You are right. I had forgotten about the rewriting. Of course the rewriting for current sqlite won't fill in the missing values like the migration scripts do.